src/WebJobs.Extensions.DurableTask/Options/DurableTaskOptions.cs

```diff
@@ -27,7 +30,27 @@ public class DurableTaskOptions
         /// multiple Durable Functions applications from each other, even if they are using the same storage backend.
         /// </remarks>
         /// <value>The name of the default task hub.</value>
-        public string HubName { get; set; }
+        public string HubName
+        {
+            get
+            {
+                if (this.hubName == null)
+                {
+                    this.isDefaultHubName = true;
+
+                    // "WEBSITE_SITE_NAME" is an environment variable used in production. When testing locally, you can set this
```

Comment:
A few comments about this comment:

- In general, we avoid using "you" to describe how customers can configure the code. I would replace that with "customers", or even using the passive voice.
- Be a bit more descriptive than "production", especially since that word is used in another context (production slots) in the same file. I would replace "production" with "Azure Functions infrastructure". 
- `local.json` should be `local.settings.json`

Reply:
Sounds good, I'll make those changes!